### ISSUE TITLE
remove all instances of pull-right

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#14544]
`pull-right` uses `float: right`, which requires elements to be out of order (first, before the left content). This breaks accessibility, both for screen readers and for keyboard navigation. Additionally, using floats to place elements is fragile.

Instead, more modern approaches should be used (and this may include using flex or grid).

We should have a PR that goes through Cockpit and fixes all instances of `pull-right`... and remove the class altogether (if it's in Cockpit and not PF3/Bootstrap).
